chart/kubeapps/README.md

```diff
@@ -106,17 +106,27 @@ apprepository:
 EOF
 $ helm install --name kubeapps --namespace kubeapps bitnami/kubeapps -f custom-values.yaml
 ```
+### Configuring connection to a custom namespace Tiller instance
 
-### Configuring connection to a secure Tiller installation
+By default, Helm installs Tiller in the `kube-system` namespace, being the default endpoint that Kubeapps tries to connect to. 
 
-In some cases, your organization will rely on a custom, secure installation of Tiller, the Helm server side component.
+If your instance of Tiller is running in a different namespace or you want to have different instances of Kubeapps connected to different Tiller instances, you can achieve it by doing:
+
+```console
+helm install \
+  --set tillerProxy.host=tiller-deploy.my-custom-namespace:44134 \
+  bitnami/kubeapps
+```
+
+### Configuring connection to a secure Tiller instance
+
+In some cases, your organization will rely on a custom, [secure installation of Tiller](https://docs.helm.sh/using_helm/#using-ssl-between-helm-and-tiller), the Helm server side component.
```

Comment:
I would make this wording a bit stronger and say we strongly recommend setting up Tiller securely when running in production